packages/client/lib/sentinel/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { RedisNode } from './types';
+
+/* TODO: should use map interface, would need a transform reply probably? as resp2 is list form, which this depends on */
+export function parseNode(data: any): RedisNode | undefined{
+  const flags: string = data[9];
+  if (flags.includes("s_down")) {
+    return undefined;
+  }
+
+  return { "host": data[3], "port": Number(data[5]) };
+}
+
+export function createNodeList(data: any) {
```

Comment:
There's a lot of any for typing here...

Reply:
yes, I dont know how to get typing out of raw redis commands yet.